packages/react-native-tracer-provider/src/EmbraceNativeTracerProvider.ts

```diff
@@ -56,13 +61,13 @@ export const useEmbraceNativeTracerProvider = (
     if (isLoading) {
       NativeModules.EmbraceManager.isStarted()
         .then((started: boolean) => {
-          if (started) {
-            setTracerProvider(new EmbraceNativeTracerProvider(config));
-          } else {
+          if (!started) {
             setError(
               "The Embrace SDK must be started to use the TracerProvider, please invoke `initialize` from `@embrace-io/react-native`.",
             );
             setIsError(true);
+          } else if (!tracerProvider) {
```

Comment:
@facostaembrace I couldn't reproduce seeing the view span recorded twice after making this change so I'll assume for now it was due to accidentally instantiating this provider multiple times